swagger/v1/api_spec.json

```diff
@@ -1391,7 +1483,8 @@
               "participant_id": "db3a7848-7308-4879-942a-c4a70ced400a",
               "declaration_type": "started",
               "declaration_date": "2021-05-31T02:21:32.000Z",
-              "course_identifier": "ecf-mentor"
+              "course_identifier": "ecf-mentor",
+              "eligible_for_payment": true
```

Comment:
That is the place that now gets `eligible_for_payment` where it does not belong, I think.